style.css

```diff
@@ -0,0 +1,226 @@
+body {
+  background-color: #eee;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  min-height: 75px;
+  padding: 0 20px;
+  display: inline-block;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #eee;
+}
+
+nav {
+  display: inline-block;
+  margin-top: 10px;
+  width: 50%;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  margin-top: 15px;
+  width: 49.5%;
+  position: relative;
+}
+
+.logo > img {
+  width: 100%;
+  height: 100%;
+  max-width: 300px;
```

Comment:
Kindly use the responsive units in line 43 because in different browsers sizes should be the same style